tests/languages/python_test.py

```diff
@@ -32,3 +32,24 @@ def test_sys_executable_matches(v):
 def test_sys_executable_matches_does_not_match(v):
     with mock.patch.object(sys, 'version_info', (3, 6, 7)):
         assert not python._sys_executable_matches(v)
+
+
+@pytest.mark.parametrize(
+    'exe,realpath,expected', (
```

Comment:
would prefer a tuple of strings here instead of stringly typed parametrize as well